packages/mask/src/plugins/RedPacket/schema.json

```diff
@@ -87,7 +87,7 @@
                     "title": "symbol"
                 }
             },
-            "required": ["address", "decimals", "name", "symbol"],
```

Comment:
`"decimals", "name", "symbol"` can be fetched later, since sheer size of payload would cause encoded image not work.